dev/src/index.ts

```diff
@@ -659,6 +659,9 @@ export class Firestore implements firestore.Firestore {
    * @returns {WriteBatch} A WriteBatch that operates on this Firestore
    * client.
    *
+   * @deprecated This class is now deprecated. Use Transaction for atomic,
```

Comment:
"Atomic and parallel" seems a little incorrect.

How about: Use `runTransaction()` to update multiple documents atomically or `bulkWriter()` to update a large number of document in parallel.


We should add this in four places:
- WriteBatch in src/
- batch() in src/
- WriteBatch in types/
- batch() in types/

Reply:
done